packages/parser/src/language/info/infoParser.ts

```diff
@@ -0,0 +1,20 @@
+import type { AstNode, LangiumParser, ParseResult } from 'langium';
+import { createLangiumParser } from 'langium';
+
+import type { InfoServices } from './infoModule.js';
+import { isInfo } from '../generated/ast.js';
+import { version } from '../../../package.json';
```

Comment:
There is no need to import this package.json. It's extra size, that's not gonna be used.